### ISSUE TITLE
A11y_code.visualstudio.com(CELA-P1)_VS Code FAQ_Screen Reader:Nvda is not reading the role and label of "follow@code"

### ISSUE THREAD

**Issue by Amlan454** [microsoft/vscode-docs#2647]
Check out Keros!- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**Environment Details:**
code.visualstudio.com/docs/supporting/faq
Browser: chrome
Screen Reader: NVDA

**Note:
This issue is observed through out the application:**
code.visualstudio.com/docs
code.visualstudio.com/download
code.visualstudio.com/docs/supporting/faq
code.visualstudio.com/docs/setup/linux
code.visualstudio.com/docs/getstarted/settings
code.visualstudio.com/docs/python/python-tutorial
code.visualstudio.com/updates/v1_30
code.visualstudio.com/docs/editor/whyvscode
code.visualstudio.com/docs/getstarted/tips-and-tricks
code.visualstudio.com/updates/v1_29
code.visualstudio.com/docs/editor/integrated-terminal
code.visualstudio.com/docs/editor/debugging
code.visualstudio.com/docs/editor/versioncontrol

**Repro Steps:**
1)Hit the URL(code.visualstudio.com/docs/supporting/faq)and login by using appropriate credential
2)Switch on the NVDA 
3)Navigate to follow@code  in the footer navigation
4)Check whether NVDA is reading it correctly or not

**Actual Result:**
Nvda is not reading the role and label of follow@code. It is reading blank

**Expected Result:**
NVDA should read the label name and role of "Follow@code"

**User Impact:**
Screen Reader users will not understand the screen reader focus landed on the which control

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by Amlan454** [comment]
[Bug_2647_NVDA is not reading the role and label.pptx](https://github.com/microsoft/vscode-docs/files/3161841/Bug_2647_NVDA.is.not.reading.the.role.and.label.pptx)

**Comment by Amlan454** [comment]
![Bug_2647_1](https://user-images.githubusercontent.com/42638683/57606413-71fd3d80-7586-11e9-9bb4-087eea364819.png)

**Comment by Amlan454** [comment]
![Bug_2647_2](https://user-images.githubusercontent.com/42638683/57606745-2ac37c80-7587-11e9-858b-358e6b69ff6d.png)

**Comment by auchenberg** [comment]
Should be fixed now, as the button isn't no longer hidden.

**Comment by fsteffi** [comment]
GitHubTags:#CAITestPass_June’19;
